File: PaMVPDemo/app/src/main/java/com/example/wen/pamvpdemo/login/LoginFragmentActivity.java
package com.example.wen.pamvpdemo.login;

import android.widget.FrameLayout;

import com.example.wen.pamvpdemo.R;
import com.pa.core.base.BaseActivity;
import com.pa.core.base.IPresenter;

import butterknife.BindView;

public class LoginFragmentActivity extends BaseActivity {

    @BindView(R.id.fl_container)
    FrameLayout flContainer;

    @Override
    protected int getlayoutId() {
        return R.layout.activity_login_fragment;
    }

    @Override
    protected IPresenter createPresenter() {
        return null;
    }

    @Override
    public void initView() {
        android.support.v4.app.FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
        fragmentTransaction.add(R.id.fl_container,new LoginFragment(),null).commit();
    }
}

File: PaMVPKotlinDemo/app/src/main/java/com/example/wen/pamvpdemo/login/LoginFragmentActivity.kt
package com.example.wen.pamvpdemo.login

import com.example.wen.pamvpdemo.R
import com.pa.core.base.BaseActivity
import com.pa.core.base.IPresenter

class LoginFragmentActivity : BaseActivity<LoginContract.LoginPresenter>(), LoginContract.LoginView {

    override fun onSuccess(any: Any) {
    }

    override fun onFailure() {
    }

    override fun getlayoutId(): Int {
        return R.layout.activity_login_fragment
    }

    override fun createPresenter(): IPresenter? {
        return null
    }

    override fun initView() {
        val fragmentTransaction = supportFragmentManager.beginTransaction()
        fragmentTransaction.add(R.id.fl_container, LoginFragment(), null).commit()
    }
}
